packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js

```diff
@@ -40,16 +40,21 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     zIndex: 2,
     alignItems: 'center',
+    height: '100%',
   },
   backIcon: {
     marginEnd: 8, // eslint-disable-line backpack/use-tokens
   },
+  backIconWithoutTitle: {
+    marginEnd: 16, // eslint-disable-line backpack/use-tokens
```

Comment:
What's the reason for not using tokens? I think there should be a comment to go with this explaining the decision...

Reply:
Sorted